packages/blade/src/components/Charts/DonutChart/DonutChart.stories.tsx

```diff
@@ -0,0 +1,289 @@
+
+const chartData = [
+  { name: 'Group A', value: 400 },
+  { name: 'Group B', value: 300 },
+  { name: 'Group C', value: 300 },
+  { name: 'Group D', value: 200 },
+  { name: 'Group E', value: 100 },
+];
+
+// 1. Basic Donut Chart
+export const BasicDonutChart: StoryFn<typeof ChartDonut> = ({ ...args }) => {
+  return (
+    <Box display="flex" justifyContent="center" alignItems="center" width="100%" height="100%">
+      <ChartDonutWrapper width="500px" height="300px">
+        <ChartDonut {...args} dataKey="value" nameKey="name" data={chartData} />
+        <ChartLegend />
+        <ChartTooltip />
+      </ChartDonutWrapper>
+    </Box>
+  );
+};
+
+// 2. Donut Chart with Center Text
+export const DonutChartWithCenterText: StoryFn<typeof ChartDonut> = () => {
+  return (
+    <Box display="flex" justifyContent="center" alignItems="center" width="100%" height="100%">
+      <ChartDonutWrapper label="Total" text="1300" width="500px" height="300px">
+        <ChartDonut dataKey="value" nameKey="name" data={chartData} radius="medium">
+          <ChartDonutCell color="chart.background.sequential.azure.600" />
+          <ChartDonutCell color="chart.background.sequential.azure.500" />
+          <ChartDonutCell color="chart.background.sequential.azure.400" />
+          <ChartDonutCell color="chart.background.sequential.azure.300" />
+          <ChartDonutCell color="chart.background.sequential.azure.200" />
+        </ChartDonut>
+        <ChartLegend />
+        <ChartTooltip />
+      </ChartDonutWrapper>
+    </Box>
+  );
+};
+
+DonutChartWithCenterText.parameters = {
+  controls: { disable: true },
+};
+
+// 3. Small Radius Donut Chart
+export const SmallRadiusDonutChart: StoryFn<typeof ChartDonut> = () => {
+  return (
+    <Box display="flex" justifyContent="center" alignItems="center" width="100%" height="100%">
+      <ChartDonutWrapper width="500px" height="500px">
+        <ChartDonut dataKey="value" nameKey="name" data={chartData} radius="small">
+          <ChartDonutCell color="chart.background.sequential.azure.600" />
+          <ChartDonutCell color="chart.background.sequential.azure.500" />
+          <ChartDonutCell color="chart.background.sequential.azure.400" />
+          <ChartDonutCell color="chart.background.sequential.azure.300" />
+          <ChartDonutCell color="chart.background.sequential.azure.200" />
+        </ChartDonut>
+        <ChartTooltip />
+      </ChartDonutWrapper>
+    </Box>
+  );
+};
+
+SmallRadiusDonutChart.parameters = {
+  controls: { disable: true },
+};
+
+// 4. Extra Large Radius Donut Chart
+export const ExtraLargeRadiusDonutChart: StoryFn<typeof ChartDonut> = () => {
+  return (
+    <Box display="flex" justifyContent="center" alignItems="center" width="100%" height="100%">
+      <ChartDonutWrapper width="500px" height="400px">
+        <ChartDonut dataKey="value" nameKey="name" data={chartData} radius="large">
+          <ChartDonutCell color="chart.background.sequential.azure.600" />
```

Comment:
If not now, when are we doing the sequantial colors?

Reply:
sequantial colors is already supported ?